doc/authentication.md

```diff
@@ -172,6 +172,83 @@ In an unrestricted project, only the `auditor` and the `user` roles are suitable
 In a {ref}`restricted project <projects-restrictions>`, the `operator` role is safe to use as well if configured appropriately.
 ```
 
+## TLS server certificate
+
+LXD supports issuing server certificates using ACME services, e.g. [Let's Encrypt](https://letsencrypt.org/).
+This can be enabled by setting the `acme.domain` and `acme.email` configuration keys.
+`acme.domain` represents the domain for which the certificate should be issued, while `acme.email` is needed for creating an account with the ACME service.
+Lastly, `acme.agree_tos` needs to be set to `true` which means one agrees to the ACME service's Terms of Service.
+Per default, LXD uses "Let's Encrypt" but this can be changed by setting `acme.ca_url`.
+
+For this feature to work, LXD needs to be reachable from both ports 80 and 443.
```

Comment:
@monstermunchkin why is this, I thought verification only needs to occur over HTTP?

Reply:
That's true, verification only needs HTTP. Port 443 isn't required but it's nice if LXD was reachable from `https://lxd.example.org/1.0` without having to specify a port. But you're right, the wording is wrong.